cmd/controller/main.go

```diff
@@ -88,6 +88,14 @@ func main() {
 		logger.Fatalf("Error building pipeline clientset: %v", err)
 	}
 
+<<<<<<< HEAD
```

Comment:
conflicts :sweat:

Reply:
I just resolved them.My bad. I forgot to push local changes after resolving conflicts.